Vungle/VungleAdapter/ALVungleMediationAdapter.m

```diff
@@ -498,7 +499,7 @@ - (void)rewardedAdDidFailToLoad:(VungleRewarded * _Nonnull)rewarded withError:(N
 - (void)rewardedAdDidPresent:(VungleRewarded * _Nonnull)rewarded
 {
     NSString *creativeIdentifier = rewarded.creativeId;
-    if ( ALSdk.versionCode >= 6150000 && [creativeIdentifier al_isValidString] )
+    if ( ALSdk.versionCode >= kALSdkVersionNeeded && [creativeIdentifier al_isValidString] )
     {
         [self.rewardedAdDelegate didDisplayRewardedAdWithExtraInfo: @{@"creative_id" : rewarded.creativeId}];
```

Comment:
We might use `creativeIdentifier` instead of `rewarded.creativeId` here, like below. Thanks. :)